app/kubernetes/views/applications/helm/helm.css

```diff
@@ -0,0 +1,5 @@
+.release-table tr {
+  display: grid;
```

Comment:
a `tr` has its own display style, and I'm not sure it's considered best practice to overwrite it. why not just have a nested div?

Reply:
im going with the assumption that the styles will be additive.
Don't want to introduce additional html (where i can get by without it).